source/overview/faq.rst

```diff
@@ -406,6 +406,74 @@ The following chart highlights the end user features of Mattermost and their sup
 * X - Full Support
 * O - Partial support
 
+High Trust Questions
+----------------------
+
+What are the trust benefits of Mattermost compared to 3rd party SaaS systems that let customers manage their own encryption keys?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ 
+**Encryption doesn't mean a 3rd party SaaS vendor can't read your data**. A 3rd party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
+ 
+For example, performing a search on message histories requires access to unencrypted messages in order to match the search term to words in your unencrypted message history.
+ 
+As another example, a customer's data encryption key is unlikely to be deployed to the mobile devices of end users; therefore, when a 3rd party system sends a push notification to an end user's mobile device, the unencrypted text is available to the 3rd party.
+ 
+In contrast, Mattermost is hosted by the customer. Not only can data be encrypted at rest and in-transit with keys generated by the customer (which no vendors every touch), unencrypted data for search and mobile notifications is handled by systems under your IT team's control. 
+
+If it’s unclear from the vendor’s documentation whether or not your data can be read, ask them directly. 
+
+**Moreover, high trust enterprises need more than encryption, they need privacy, total data ownership, auditability, and control of their infrastructure.**
```

Comment:
Recommend a new question here to break up the text:  
What options for ensuring high trust does Mattermost offer?